src/CommunityToolkit.Maui/Behaviors/Validators/ValidationBehavior.shared.cs

```diff
@@ -104,7 +104,7 @@ public abstract class ValidationBehavior : BaseBehavior<VisualElement>, IDisposa
 	/// <summary>
 	/// Initialize a new instance of ValidationBehavior
 	/// </summary>
-	public ValidationBehavior() => DefaultForceValidateCommand = new Command(async () => await ForceValidate().ConfigureAwait(false));
+	protected ValidationBehavior() => DefaultForceValidateCommand = new Command(async () => await ForceValidate().ConfigureAwait(false));
```

Comment:
Breaking Change

Making a `public` API `protected`

Reply:
this is not a breaking change. class is abstract